src/app/category-chart/overview/category-chart-overview-sample.component.html

```diff
@@ -0,0 +1,19 @@
+<div>
+	<span>Chart Type:</span>
+	<select [(ngModel)]="chartType">
+		<option>Auto</option>
+		<option>Line</option>
+		<option>Column</option>
+		<option>Point</option>
+		<option>Spline</option>
+		<option>SplineArea</option>
+		<option>StepArea</option>
+		<option>StepLine</option>
+		<option>Waterfall</option>
+	</select>
+</div>
+<igx-category-chart height="500px" width="100%"
+	[dataSource]="data"
+	[chartType]="chartType"
+	isVerticalZoomEnabled="false">
```

Comment:
I don't think it makes sense to turn this of for catChart. Please remove:

```html
isVerticalZoomEnabled="false"
```

Reply:
Ah, yeah. I meant to remove it, it just annoys me while testing for some reason.